gateway/src/apicast/loader.lua

```diff
@@ -0,0 +1,51 @@
+local loadfile = loadfile
```

Comment:
I think It would be good to document the purpose of this module.

Reply:
True, good point :)

Reply:
Should be addressed in https://github.com/3scale/apicast/pull/486/commits/b1d677a043724e3b6069b35f36ae85b3e721f08d.